velox/functions/prestosql/aggregates/SumAggregate.h

```diff
@@ -244,9 +244,12 @@ bool registerSumAggregate(const std::string& name) {
           .argumentType("double")
           .build(),
       exec::AggregateFunctionSignatureBuilder()
+          .integerVariable("a_precision")
+          .integerVariable("a_scale")
+          .integerVariableWithConstraint("r_precision", "38")
           .argumentType("DECIMAL(a_precision, a_scale)")
-          .intermediateType("DECIMAL(38, a_scale)")
-          .returnType("DECIMAL(38, a_scale)")
+          .intermediateType("DECIMAL(r_precision, a_scale)")
```

Comment:
Previous code was easier to read. Can we keep it?

Reply:
Not so tricky it turns out. Done!

Reply:
Nice. Thank you, Deepak.